libreoffice-core/sw/source/filter/ww8/wrtw8nds.cxx

```diff
@@ -1434,7 +1430,7 @@ int SwWW8AttrIter::OutAttrWithRange(const SwTextNode& rNode, sal_Int32 nPos)
                         // If the hyperlink is one of our custom ones we want to unset the style of the hyperlink and break early before adding the hyperlink attributes to the document
                         const SwFormatINetFormat *rINet = static_cast< const SwFormatINetFormat* >( pItem );
                         OUString aDestinationURL = rINet->GetValue();
-                        if(aDestinationURL.startsWith(termUrl) || aDestinationURL.startsWith(termRefUrl) || aDestinationURL.startsWith(sectionUrl) || aDestinationURL.startsWith(sectionRefUrl)){
+                        if(aDestinationURL.startsWith(m_rExport.AttrOutput().termUrl) || aDestinationURL.startsWith(m_rExport.AttrOutput().termRefUrl) || aDestinationURL.startsWith(m_rExport.AttrOutput().sectionUrl) || aDestinationURL.startsWith(m_rExport.AttrOutput().sectionRefUrl)){
```

Comment:
I have a feeling you felt like this was an awkward way to get what should be a constant. If you have a gut feeling like that and you're not sure, feel free to ask me.